lib/builder.js

```diff
@@ -93,6 +93,35 @@ class MessageBuilder {
         return this;
     }
 
+    /**
+     * Set the embedded url
+     * @param {string} url The url 
+     */
+    setURL(url) {
+        this.data.attachments[0].url = url;
+        return this;
+    }
+
+    /**
+     * Set the author of the embedded object
+     * @param {string} author the author
+     * @param {string} iconURL Optional: the icon url
+     * @param {string} url Optional: the url
+     */
```

Comment:
The docs here could be nicer:
```suggestion
    /**
     * Set the author of the embed
     * @param {string} author Name of the author
     * @param {string} iconURL Optional: Icon URL of the author
     * @param {string} url Optional: Link to the author
     */
```